WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController+QuickActions.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import WordPressShared
+
+// MARK: - Quick Actions
+
+extension BlogDashboardViewController {
+
+    func showStats() {
+        trackQuickActionsEvent(.statsAccessed)
+
+        let controller = StatsViewController()
+        controller.blog = blog
+        controller.navigationItem.largeTitleDisplayMode = .never
+        showDetailViewController(controller, sender: self)
+
+        QuickStartTourGuide.shared.visited(.stats)
+    }
+
+    func showPostList() {
+        trackQuickActionsEvent(.openedPosts)
+
+        let controller = PostListViewController.controllerWithBlog(blog)
```

Comment:
Same as before.

We can have a `PostListViewController.present(for: blog, in: controller)`

Reply:
Fixed! https://github.com/wordpress-mobile/WordPress-iOS/commit/edc0ac107363c8675579937984d107989a8260db